### DISCUSSION TITLE
how use  BROWSER = 0

### DISCUSSION THREAD

**Discussion by WingDust** [terrastruct/d2#732] [Q&A] [answered]
in Windows 10 
i try
>  d2 --watch test.d2
>  d2 --watch test.d2  BROWSER = 0 
>  d2 --watch test.d2  --browser = 0

and `set BROWSER = 0` in cmd.exe

but not work

**Comment by alixander** [comment, marked as answer]
can you try `BROWSER=0 d2 --watch test.d2`

**Reply by WingDust** [reply]
![image](https://user-images.githubusercontent.com/45911286/215126527-9eb70b2d-abf6-4004-a9a6-aef75d25317a.png)
not well

**Reply by WingDust** [reply]
with d2 version  `v0.1.6`

**Reply by alixander** [reply]
@nhooyr any ideas?

**Reply by nhooyr** [reply]
I don't remember the syntax for cmd.exe or powershell. I would suggest googling how to set an environment variable for whichever you are using @WingDust, then set `BROWSER` to `0` and run the `d2` command with the appropriate arguments.

Or install and use cygwin so you can use POSIX syntax instead.

**Reply by nhooyr** [reply]
Sorry not cygwin but MSYS2. See https://github.com/terrastruct/d2/blob/master/docs/INSTALL.md#release-archives

**Reply by WingDust** [reply]
my environment variable problem, test  environment variable `BROWSER=0` is work
@nhooyr @alixander
